src/source/reference/get-feed-import-id-detail.rst

```diff
@@ -1,9 +1,107 @@
 .. index:: GET /feed/import/{id}/detail
+
 .. _get_feed_import_id_detail:
 
 GET /feed/import/{id}/detail
 ============================
 
+:ref:`get_feed_import_id_detail_v2` | :ref:`get_feed_import_id_detail_v1`
+
+.. _get_feed_import_id_detail_v2:
+
+GET /feed/import/{id}/detail V2
+-------------------------------
+
+.. list-table::
+ :widths: 20 80
+
+ * - Scope
+   - ``api_ro`` or ``console_ro``
+
+ * - Accept
+   - ``application/sellside.feedimportdetail-v2+json, application/json``
+
+This endpoint provides a summary of the feed import. Use this endpoint to get information on whether there were any
+errors or warnings associated with a specific import.
```

Comment:
can we put here the main differences wrt v1?

Reply:
Honestly, I even want to remove v1 from the docs altogether, as we don't have it implemented anymore.. what do you say?

Reply:
yeah, good idea